<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceALU</class>
 <widget class="QMainWindow" name="InterfaceALU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterfaceALU</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>631</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagen.qrc">:/Imagenes/Interfaz ALU.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_HEXA2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background:transparent;
color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Número 2 Hexadecimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HEXA1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Número 1 Hexadecimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HEXARESULT">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Resultado Hexadecimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RESULT">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>105</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_N1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_N2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AcercaDe">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;
background:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>ACERCA DE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Calcular">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Color:rgb(255, 255, 255);
font: 13pt &quot;Andale Mono&quot;;
background:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>CALCULAR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Suma">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Multipliccacion">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Division">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
